dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -483,6 +489,37 @@ twine upload -r pypi ${AIRFLOW_REPO_ROOT}/dist/*
 
 * Confirm that the packages are available under the links printed and look good.
 
+> [!NOTE]
+> The next steps until "Add tags in git" is part of the future Trusted Publishing process which we dry-run
+
+* Move and publish the `PyPI` packages to SVN
+
+```shell script
+# First clone the repo if you do not have it
+cd ..
+[ -d asf-dist ] || svn checkout --depth=immediates https://dist.apache.org/repos/dist asf-dist
+svn update --set-depth=infinity asf-dist/dev/airflow
+
+# Create a new folder for the release.
+mkdir -p asf-dist/dev/airflow/providers/pypi-rc
+cd asf-dist/dev/airflow/providers/pypi-rc
+
+# Remove previously released providers
+svn rm *
+
+# Move the artifacts to svn folder
+mv ${AIRFLOW_REPO_ROOT}/dist/* .
+
+# Add and commit
+svn add *
```

Comment:
This will not work.
We already moved from dist to release the packages to PyPi. When you get to this step dist is empty.

Reply:
Actually not. We moved the "SVN" packages to SVN, the "PyPI"  packages are created again few lines before:

```
breeze release-management prepare-provider-packages \
--version-suffix-for-pypi rc1 --package-format both PACKAGE PACKAGE ....
```

Those will have `rc1` extension unlike SVN packages which might become the final release.

BTW. We should likely change names of those "types" of packages - previously for release candidates we commited only "SVN" packages to "SVN" and "PyPI" packages for PyPI - but now both will be committed to SVN. 

So we likely need new names. Maybe:

* "no-suffix" packages
* "suffix" packages

?